std-bits/table/src/main/java/org/enso/table/data/table/aggregate/AggregateColumnDefinition.java

```diff
@@ -0,0 +1,66 @@
+package org.enso.table.data.table.aggregate;
+
+import org.enso.table.aggregations.*;
+import org.enso.table.data.table.Column;
+
+public class AggregateColumnDefinition {
```

Comment:
I don't think these aliases are really necessary, we can call the constructors from Enso by `ClassName.new`.

Unless you think it is worth keeping them to have all builders in one place - I guess it is a good enough reason.